lib/ansible/modules/system/mksysb.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: Kairo Araujo (@kairoaraujo)
+module: mksysb
+short_description: Generates AIX mksysb rootvg backups.
+description:
+  - This module manages a basic AIX mksysb (image) of rootvg.
+version_added: "2.5"
+options:
+  backup_crypt_files:
+    description:
+      - Backup encrypted files.
+    choices: ["yes", "no"]
+    default: "yes"
+  backup_dmapi_fs:
+    description:
+      - Back up DMAPI filesystem files.
+    choices: ["yes", "no"]
+    default: "yes"
+  create_map_files:
+    description:
+      - Creates a new MAP files.
+    choices: ["yes", "no"]
+    default: "no"
+  exclude_files:
+    description:
+      - Excludes files using /etc/rootvg.exclude.
```

Comment:
The file path [should be](http://ansible-docs.readthedocs.io/zh/stable-2.0/rst/developing_modules.html#module-checklist) decorated by `C()`. The same for `software_packing`.